docs/docbook5/en/output/chunkhtml/TarTask.html

```diff
@@ -5,7 +5,7 @@
         <div class="table"><a name="idp17364"></a><p class="formal-object-title"><span class="label">Table C.131: </span><span class="title">Attributes</span></p><div class="table-contents">
             
             <table summary="Attributes" border="1"><colgroup><col class="name"><col class="type"><col class="description"><col class="default"><col class="required"></colgroup><thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Default</th><th>Required</th></tr></thead><tbody><tr><td><code class="literal">destfile</code></td><td><code class="literal">String</code></td><td>Tarball filename</td><td>none</td><td>Yes</td></tr><tr><td><code class="literal">basedir</code></td><td><code class="literal">String</code></td><td>Base directory to tar (if no fileset specified, entire directory
-                            contents will be included in tar)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">compression</code></td><td><code class="literal">String</code></td><td>Type of compression to use (gzip, bzip2, none)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">includeemptydirs</code></td><td><code class="literal">Boolean</code></td><td>If set to <code class="literal">true</code>, also empty directories are copied. </td><td><code class="literal">true</code></td><td>No</td></tr><tr><td><code class="literal">longfile</code></td><td><code class="literal">String</code></td><td>How to handle long files, those with a path &gt; 100 chars. Allowable
+                            contents will be included in tar)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">compression</code></td><td><code class="literal">String</code></td><td>Type of compression to use (gzip, bzip2, lzma2, none)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">includeemptydirs</code></td><td><code class="literal">Boolean</code></td><td>If set to <code class="literal">true</code>, also empty directories are copied. </td><td><code class="literal">true</code></td><td>No</td></tr><tr><td><code class="literal">longfile</code></td><td><code class="literal">String</code></td><td>How to handle long files, those with a path &gt; 100 chars. Allowable
```

Comment:
Sorry, should have been more clear! Please add to the source file, `docs/docbook5/en/source/appendixes/optionaltasks.xml`.